bin/deploy-release.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+set -eo pipefail
+
+# Strip the 'v' from the Tag Name
+export TAG=${TAG_NAME:1}
+
+# Deploy release to OSSRH
```

Comment:
You should add a comment about what OSSRH is

Reply:
Agreed - added some more info there and in the readme